scripts/azure/index.js

```diff
@@ -166,16 +165,16 @@ class HTTPClient {
   constructor(context) {
     this.context = context;
     this.httpOptions = {
-      hostname: DD_HTTP_URL,
-      port: DD_HTTP_PORT,
+      hostname: KF_URL,
+      port: KF_HTTP_PORT,
       path: "/api/v2/logs",
       method: "POST",
       headers: {
         "Content-Type": "application/json",
-        "DD-API-KEY": DD_API_KEY,
-        "DD-EVP-ORIGIN": "azure",
+        "KF-EVP-ORIGIN": "azure",
+        "KF_API_KEY_ENV": KF_API_KEY,
```

Comment:
Don't think this will work.
This comes in as DD payload, correct?
Ingester uses DD-API-KEY.

Reply:
Yes. Okay.